examples/chart/teleport-cluster/tests/__snapshot__/auth_clusterrolebinding_test.yaml.snap

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I'm not sure why this is blank? I guess there's no snapshotting needed, so rather than have no file it just has an empty object?